mkdocs.yml

```diff
@@ -32,7 +32,8 @@ nav:
 - Contributing to the docs: contributing-docs.md
 
 theme:
-  name: 'readthedocs'
+    name: readthedocs
+    custom_dir: docs-theme-overrides/
```

Comment:
Per [the docs](https://www.mkdocs.org/user-guide/configuration/#custom_dir), this can be a directory relative to the MkDocs configuration file. If I move the overrides dir to `docs/theme-overrides` and change this here to the same (either `docs/theme-overrides` or `./docs/theme-overrides`), it works for me locally (`mkdocs serve`) without cluttering up the repository root.

I see your comment that it cannot be nested; is there some other way of serving and building that doesn't work with a relative directory?

Reply:
doh, i thought i tried it but i had forgotten to update the file path in the config here. thanks, i'll push this change.